Ibro.tsx

```diff
@@ -0,0 +1,187 @@
+import React, { Component } from "react";
+import { Container, Button, Text, View } from "native-base";
+////import { Col, Row, Grid } from "react-native-easy-grid";
+import { StyleSheet, TextInput, TouchableOpacity } from "react-native";
+
+class Inputs extends Component {
```

Comment:
Why is this file called 'Ibro'? Change it to an appropriate name.

Reply:
to find myself. I might need his design.

Reply:
Will ignore it.